src/pages/home/report/withReportOrNotFound.tsx

```diff
@@ -36,7 +36,7 @@ export default function (
             const isReportIdInRoute = props.route.params.reportID?.length;
 
             if (shouldRequireReportID || isReportIdInRoute) {
-                const shouldShowFullScreenLoadingIndicator = props.isLoadingReportData && (!Object.entries(props.report ?? {}).length || !props.report?.reportID);
+                const shouldShowFullScreenLoadingIndicator = props.isLoadingReportData !== false && (!Object.entries(props.report ?? {}).length || !props.report?.reportID);
```

Comment:
Sorry, why this change was made?

Reply:
@MonilBhavsar  When I took merge a week back, I found that someone made a regression when renaming the file from js to tsx.
In [old withReportOrNotFound.js](https://github.com/Expensify/App/blob/9fb902b6e1fa6316db3b91dd4dac51d7b6972aa5/src/pages/home/report/withReportOrNotFound.js#L44), we had default props setting isLoadingReportData to true but the default props were removed in tsx file.

This change is required when default props is not provided. This changes fixes the regression caused by removing default props at all places which uses withReportOrNotFound.tsx.

What I propose is to unblock this PR with this change and then revisiting on how to bring back the default props in tsx. I think adding back default props should be out of scope of this PR. What are your thoughts on this?